lldb/test/API/lang/swift/implementation_only_imports/library_resilient/TestLibraryResilient.py

```diff
@@ -82,7 +82,7 @@ def cleanup():
         # This test is deliberately checking what the user will see, rather than
         # the structure provided by the Python API, in order to test the recovery.
         self.expect("fr var", substrs=[
-            "(SomeLibrary.ContainsTwoInts) container = (other = 10)",
+            "(SomeLibrary.ContainsTwoInts) container = {", "other = 10",
```

Comment:
This is perhaps indicative of issues that need to be solved in this PR, or in follow ups.

`ContainsTwoInts` is defined as:

```swift
public struct ContainsTwoInts {
  internal var wrapped: TwoInts
  public var other: Int
}
```

This was previously formatted as `container = (other = 10)` but with these changes it's formatted as:

```
container = {
  other = 10
}
```

The same data is printed, but the difference is (presumably) that `ShouldPrintAsOneLiner` is no longer true. Note that despite there being two children in the type, lldb only prints only one, the public property `other`.

Based on my debugging of other discrepancies, my guess is that `GetNumChildren` now returns 2 instead of 1, which then causes `ShouldPrintAsOneLiner` to return false.

What I think should be possible now, is for lldb to print both properties, `wrapped` and `other`. I'll take a look tomorrow to see if the `TypeRef` implementation of `GetChildCompilerTypeAtIndex` is excluding `wrapped` to mirror the behavior of `SwiftASTContext`. If that is the case, it seems like something we should fix, showing private properties too would be an improvement.

Reply:
Sounds great! Thanks for the thoughtful analysis.